src/Umbraco.Web/Models/Mapping/CodeFileDisplayMapper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
+using Umbraco.Core;
+using Umbraco.Core.Models.Mapping;
+using Umbraco.Core.Models;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Web.Models.Mapping
+{
+    public class CodeFileDisplayMapper : MapperConfiguration
+    {
+        public override void ConfigureMappings(IConfiguration config, ApplicationContext applicationContext)
```

Comment:
Assuming you've run the `AutoMapperTests` to make sure the correct ignore's are in place?